src/haggadah/views.cljs

```diff
@@ -215,32 +212,42 @@ To see changes in preview edit source and then click on preview.
 
 (defn haggadah-creation-panel
   []
-  (let [text (atom "## The best possible Haggadah")]
+  (let [haggadah (atom nil)]
     [:div.pt-4.page 
      [:div.hero.is-fullheight
       [:div.columns.is-centered
        [:div.column.is-5-tablet.is-4-desktop.is-3-widescreen
         [:form.box.mt-4 
-         [:h1.pb-4 "Please fill in the details of your Haggadah below"]
+         [:h1.pb-4 "Please make a title and select the haggadah template"]
```

Comment:
Maybe _enter a title_ is a bit more clear.

Reply:
I'll change the message so it asks for the user to enter a title.